tests/playwright-tests/utils/env.ts

```diff
@@ -3,8 +3,17 @@ import generateRunIdentifier from './generateRunIdentifier';
 
 dotenv.config();
 
+const publicUrl =
```

Comment:
Ahh did you decide to go with this approach in the end?  I know that we thought it might be better if we could add the Authorisation header to just the public site URLs using extraHTTPHeaders but only on particular routes.  Did you get a chance to look into that?

Reply:
FYI I had a look at what we do in the Robot Tests and it looks as though we don't expose the basic auth values on the URLs in the test report (or anywhere else in the test report).

I think it'd be nice to try to do the same here with the suggestion that you had (where we only add the extra HTTP headers on the public routes).  Is this something you could try out?